server/src/main/java/io/deephaven/server/table/inputtables/InputTableServiceGrpcImpl.java

```diff
@@ -40,6 +40,7 @@ public InputTableServiceGrpcImpl(TicketRouter ticketRouter, SessionService sessi
     public void addTableToInputTable(AddTableRequest request, StreamObserver<AddTableResponse> responseObserver) {
         GrpcUtil.rpcWrapper(log, responseObserver, () -> {
             final SessionState session = sessionService.getCurrentSession();
+            session.getAuthContext().requirePrivilege(InputTableServicePrivilege.CAN_ADD_TO_INPUT_TABLE);
```

Comment:
I'm mentioning it here - but it's a much more widely applicable comment.

How do we enable authorization based on the contents of the request?

For example, I want "devin" to be able to add to the input table `devins_table`, but not `nates_table`.